### PR TITLE
Fix multiline streets not getting parsed properly

### PR DESCRIPTION
**Pull request by tdgroot**
We were receiving the following error:

```
An error has occurred while validating order number <curated>. Check address.-MyParcelNL\Sdk\src\Exception\InvalidConsignmentException: Something went wrong splitting up the following address: Some Street
123
5 A
```

### CODE REVIEW

File: src/Helper/SplitStreet.php [hunk 1]
```diff
@@ -77,6 +77,8 @@ public static function splitStreet(string $fullStreet, string $local, string $de
             foreach (self::NUMBER_SUFFIX_ABBREVIATION as $from => $to) {
                 $fullStreet = preg_replace("/(\d.*-?)[\s]$from/", '$1' . $to, $fullStreet);
             }
+
+            $fullStreet = trim(preg_replace('/(\r\n)|\n|\r/', ' ', $fullStreet));
```
Comment (reindert-vetter) [inline src/Helper/SplitStreet.php:81]: I think it's good to add a test to the split street tests.
Or you can of put this logic in a function. Than you can make a unit test for that function.

**Review by reindert-vetter — CHANGES_REQUESTED** [review]

File: src/Helper/SplitStreet.php (hunk 1 above)
Comment (reindert-vetter) [inline src/Helper/SplitStreet.php:81]: I see the same thing happening on line 92. You can take that away. If you took away that line, I will accept your pull-request.

**Review by reindert-vetter — CHANGES_REQUESTED** [review]

File: src/Helper/SplitStreet.php (hunk 1 above)
Comment (tdgroot) [reply src/Helper/SplitStreet.php]: Duplicate code removed.

**Comment by reindert-vetter** [comment]
I think the adjustment could be simpler. I have created a pull request: https://github.com/myparcelnl/sdk/pull/284/files
Would you like to check this PR?

**Comment by reindert-vetter** [comment]
Fixed in https://github.com/myparcelnl/sdk/pull/284/files

**Pull request closed without merging**